plugin.yaml

```diff
@@ -1,11 +1,11 @@
 name: "ssm"
-version: "1.0.10"
+version: "dev"
```

Comment:
Is this intentional? Or do we want to bump our major version instead?

Reply:
This gets updated when packaging (cf. https://github.com/codacy/helm-ssm/blob/7fdcf4e3c3521f201c66fbc6373d324aec6b665d/Makefile#L34 ). Hence, it didn't seem nice to have meaningless version number hardcoded here. Moreover, if you don't have a version number it is most likely because you're compiling from source for development purposes, so `dev` seems to fit the bill.